Extensions/Kayenta/Src/Tasks/createCanaryAnalysisRequestV0/task.json

```diff
@@ -0,0 +1,298 @@
+    },
+    {
+      "name": "thresholds",
+      "displayName": "Thresholds",
+      "isExpanded": true
+    },
+    {
+    "name": "executionRequest",
+    "displayName": "Analysis Configuration",
+    "isExpanded": false
+    },
+    {
+    "name": "scopes",
+    "displayName": "Scopes",
+    "isExpanded": false
+    }
+],
+  "inputs": [
+    {
+        "name": "KayentaConnection",
+        "type": "connectedService:kayenta",
+        "label": "Kayenta connection",
+        "defaultValue": "",
+        "required": "true",
+        "helpMarkDown": "Connection to the Kayenta instance used for canary analysis."
+    },
+    {
+      "name": "metricsAccountName",
+      "type": "string",       
+      "label": "Metric account",
+      "required": "true",
+      "helpMarkDown": "Name of metric account configured for Kayenta Ex : datadog-account",
+      "groupName": "accountConfiguration",
+      "properties": {
+          "EditableOptions": "True"
+        }
+    },
+    {
+      "name": "storageAccountName",
+      "type": "string",       
+      "label": "Storage account",
+      "required": "true",
+      "helpMarkDown": "Name of storage account configured for Kayenta Ex : google-account",
+      "groupName": "accountConfiguration",
+      "properties": {
+          "EditableOptions": "True"
+        }
+    },
+    {
+      "name": "configAccountName",
+      "type": "string",      
+      "label": "Configuration Account",
+      "required": "true",
+      "helpMarkDown": "Configuration account name to fetch canary configurations. Ex : google-account",
+      "groupName": "accountConfiguration",
+      "properties": {
+        "EditableOptions": "True"
+        }
+    },
+    {
+      "name": "canaryConfig",
+      "type": "pickList",      
+      "label": "Canary config",
+      "required": "true",
+      "helpMarkDown": "Choose the canary-config for the analysis.",
+      "groupName": "accountConfiguration",
+      "properties": {
+        "EditableOptions": "False"
+        }
+    },
+    {
+        "name": "marginalThreshold",
+        "type": "string",       
+        "label": "Marginal threshold",
+        "required": "false",
+        "defaultValue": "0",
+        "helpMarkDown": "Canary classification will return Marginal if canary-score falls above marginal threshold else return Fail, Ex : 75.0",
+        "groupName": "thresholds",
+        "properties": {
+            "EditableOptions": "True"
+          }
+    },
+    {
+        "name": "passThreshold",
+        "type": "string",      
+        "label": "Pass threshold",
+        "required": "true",
+        "helpMarkDown": "Canary will Pass only if canary-score crosses pass threshold, Ex : 95.0",
+        "groupName": "thresholds",
+        "properties": {
+            "EditableOptions": "True"
+          }
+    },
+    {
+        "name": "lifetimeDurationMins",
+        "type": "string",      
+        "label": "Analysis lifetime duration {mins}",
+        "required": "true",
```

Comment:
Change {mins}. Square brackets are not looking nice. Just check once how it is mentioned in other tasks. Also add validation for this input.